explorer/testdata/destructor/destructor_in_destructor_calls.carbon

```diff
@@ -0,0 +1,58 @@
+// Part of the Carbon Language project, under the Apache License v2.0 with LLVM
```

Comment:
For filenames, can you omit the "destructor_in" in general? Part of the reason for directories is to make it implicit that everything inside is testing destructors, without per-filename reminders. :)